services/autoscaling/src/simcore_service_autoscaling/api/health.py

```diff
@@ -22,13 +22,22 @@ async def health_check():
     return f"{__name__}.health_check@{datetime.utcnow().isoformat()}"
 
 
-class _RabbitMQStatus(BaseModel):
+class _ComponentStatus(BaseModel):
     initialized: bool
```

Comment:
MINOR:  I understand that the first field is used when the client is init and the second if the API is responsive (i.e. respond s to a ping)

what about ``is_enabled`` and ``is_responsive`` instead ??